client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -111,23 +114,33 @@ class Slideshow extends Component {
 				>
 					<ul className="wp-block-jetpack-slideshow_swiper-wrappper swiper-wrapper">
 						{ images.map( ( { alt, caption, id, url } ) => (
-							<li className="wp-block-jetpack-slideshow_slide swiper-slide" key={ id }>
-								<figure>
-									<img
-										alt={ alt }
-										className={
-											`wp-block-jetpack-slideshow_image wp-image-${ id }` /* wp-image-${ id } makes WordPress add a srcset */
-										}
-										data-id={ id }
-										src={ url }
-									/>
-									{ caption && (
-										<figcaption className="wp-block-jetpack-slideshow_caption gallery-caption">
-											{ caption }
-										</figcaption>
+							( ! forSave || ( forSave && ! isBlobURL( url ) ) ) && (
```

Comment:
Wondering if we should really exclude the newly added image from saving to markup at block level. As an alternative, can we disable the Save button (and autosave) in the editor while images are uploading?

Reply:
That's possible, although I have concerns about taking blocking save. A network problem, invalid image, or other upload shenanigans could end up blocking save indefinitely for a post, which could cause confusion and even data loss. Does that concern you?

Reply:
Yeah, good question. I wonder if Gutenberg/core has made any provisions for those cases when disabling Save...
A quick Google search didn't even turn up anything on how to disable Save at all. I'll ask in Core Slack quickly, and if there isn't a good way, let's go with this PR.

Reply:
https://wordpress.slack.com/archives/C02QB2JS7/p1551107657256400

Reply:
Also, as I noted [here](https://github.com/Automattic/wp-calypso/issues/30920#issuecomment-466478691) this problem can be reproduced in various ways in Core Gallery and Tiled Gallery, so there doesn't seem to be an established approach for handling yet.

Reply:
@aduth thankfully pointed me to [`lockPostSave`](https://wordpress.org/gutenberg/handbook/designers-developers/developers/data/data-core-editor/#lockpostsaving).

> I have concerns about taking blocking save. A network problem, invalid image, or other upload shenanigans could end up blocking save indefinitely for a post, which could cause confusion and even data loss

He also confirmed that core was having the same problem wrt locking post save, pointing me specifically to https://github.com/WordPress/gutenberg/issues/8119, which is about solving issues related to Undo-ing that sort of state at a framework level.

My takeaway is that we wouldn't be 'worse than core' if we used `lockPostSave`, and that we potentially profit from a framework-level solution as soon as there is one.

However, I'm also fine with keeping this block-specific solution -- I'd just add a code comment providing some rationale (esp so we might find it more easily if we ever are to update the block to use a framework level solution later one).